code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5549,3 +5549,108 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	reagent_state = SOLID
 	color = "#FFA500"
 	custom_metabolism = 0.1
+
+//Plant-specific reagents
+
+/datum/reagent/kelotane/tannic_acid
+	name = "Tannic Acid"
+	id = TANNIC_ACID
+	description = "Tannic acid is a natural burn remedy."
+	reagent_state = LIQUID
+	color = "#C8A5DC" //rgb: 200, 165, 220
+
+/datum/reagent/dermaline/kathalai
+	name = "Kathalai"
+	id = KATHALAI
+	description = "Kathalai is an exceptional natural burn remedy, it performs twice as well as tannic acid."
+	color = "#C8A5DC" //rgb: 200, 165, 220
+
+/datum/reagent/bicaridine/opium
+	name = "Opium"
+	id = OPIUM
+	description = "Opium is an exceptional natural pain reliever."
```

Comment:
Bicaridine is stated to be an analgesic (painkiller) in its description, yet it heals brute damage.

I'm not sure there are any chemicals whatsoever, let alone natural chemicals, that actually repair tissue damage.